data/json/items/comestibles/med.json

```diff
@@ -1507,6 +1515,7 @@
     "healthy": -2,
     "addiction_potential": 10,
     "addiction_type": "opiate",
+    "vitamins": [ [ "vit_mme", 200 ] ],
```

Comment:
same thing, 1mg of morphine should be 100vit_mme ?

Reply:
You have to read the comment in each item as to what is being referenced.  All the mutated poppy items are 1mg of morphine per ml of item.

Reply:
Honestly this item shouldn't be removing any euphoria or drowsiness.  But that's likely another PR.

Reply:
oh, i see, the sentence structure in the comment confused me (i misread it as "[this is] 1 ML equivalent [of morphone. I.e. equivalent to 1ml of morhpne. Which is the same as] 1 mg of morphine"). I get it now.

Reply:
Hopefully the adjustments I made to the comment make that clearer :) feel free to suggest changes if needed.